core/sail/lmdb/src/main/java/org/eclipse/rdf4j/sail/lmdb/LmdbRecordIterator.java

```diff
@@ -23,14 +23,15 @@
 import org.eclipse.rdf4j.sail.lmdb.Varint.GroupMatcher;
 import org.lwjgl.PointerBuffer;
 import org.lwjgl.system.MemoryStack;
-import org.lwjgl.system.MemoryUtil;
 import org.lwjgl.util.lmdb.MDBVal;
 
 /**
  * A record iterator that wraps a native LMDB iterator.
  *
  */
 public class LmdbRecordIterator implements RecordIterator {
```

Comment:
I realize I bang on a lot about this, but: can we make this (and a few other classes that are really not intended for external use) package-private? If you can't because there's multiple packages that need access, that's fine - perhaps mark it `@InternalUseOnly` then.